esbuild.config.mjs

```diff
@@ -0,0 +1,35 @@
+import chalk from "chalk";
+import { analyzeMetafile, build } from "esbuild";
+
+(async () => {
+  try {
+    const startTime = Date.now();
+    console.info(
+      chalk.bold(`🚀 ${chalk.blueBright("gh-node-module-generation")} Build\n`)
+    );
+
+    const result = await build({
+      entryPoints: ["./src/index.js"],
+      outfile: "dist/index.js",
+      metafile: true,
+      bundle: true,
+      platform: "node",
+      target: ["node16"],
```

Comment:
please add a comment here, that the version must be consistent with the version used for the gh-action.